test/xpu_api/utilities/pairs/pairs.pair/noexcept_swap_g.pass.cpp

```diff
@@ -0,0 +1,85 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/utility>
+
+#include "support/utils.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+struct NoexceptMoveAssignClass                          // KSATODO (empty)
+{
+};
+
+struct NoexceptMoveConsClass                            // KSATODO (empty)
+{
+};
+
+// KSATODO struct NoexceptMoveConsNoexceptMoveAssignClass
+struct NoexceptMoveConsNoexceptMoveAssignClass
+{
+};
+
+template <typename T>
+void 
+test_is_nothrow_swappable()
+{
+    static_assert(std::is_nothrow_swappable<T>::value);
+    static_assert(std::is_nothrow_swappable_v<T>);
+
+    static_assert(std::is_nothrow_swappable_with<T, T>::value);
+    static_assert(std::is_nothrow_swappable_with_v<T, T>);
```

Comment:
Since we are testing swap here, not `is_nothrow_swappable` trait, I guess only one check is enough:
```suggestion
    static_assert(std::is_nothrow_swappable_v<T>);
```

Also in this case we can simply change `test_is_nothrow_swappable` function calls to direct call to `std::is_nothrow_swappable_v`.